src/App.jsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
Do not forget to
<img width="256" alt="Screenshot 2022-09-16 at 15 09 11" src="https://user-images.githubusercontent.com/17098736/190635692-70951961-2599-41d6-beb2-677f29264024.png">
 check your task with a [checklist ](https://github.com/mate-academy/react_sum/blob/master/checklist.md) and put the mark in the table.